docs/cep/query-guide/query.md

```diff
@@ -495,7 +495,7 @@ The syntax of aggregate function is as follows,
 ```
 insert into <output stream>
 select <aggregate function>(<parameter>, <parameter>, ... ) as <attribute name>, <attribute2 name>, ...
-from <input stream>#window.<window name>(<parameter>, <parameter>, ... );
+from <input stream> window <window name>(<parameter>, <parameter>, ... );
 ```
 
 Here `<aggregate function>` uniquely identifies the aggregate function. The `<parameter>` defined input parameters the aggregate function can accept. The input parameters can be attributes, constant values, results of other functions or aggregate functions, results of mathematical or logical expressions, or time values. The number and type of parameters an aggregate function accepts depend on the function itself.
```

Comment:
```suggestion
Here `<aggregate function>` uniquely identifies the aggregate function. The `<parameter>` defines input parameters the aggregate function can accept. The input parameters can be attributes, constant values, results of other functions or aggregate functions, results of mathematical or logical expressions, or time values. The number and type of parameters an aggregate function accepts depend on the function itself.
```

Reply:
If you have more than three options in a list, such as in the third sentence, then it should probably be an unordered (bulleted) list. Change the format to be a list, do not forget to remove punctuation.